tests/foreman/api/test_acs.py

```diff
@@ -116,6 +116,7 @@ def test_positive_run_bulk_actions(module_target_sat, module_yum_repo):
     :steps:
         1. Create several ACSes.
         2. Bulk refresh them all.
+        3. Add 1 more ACS, then use bulk refresh all.
         3. Bulk destroy some of them.
```

Comment:
This should fix `make docs` in CI/CQ
```suggestion
        4. Bulk destroy some of them.
        5. Cleanup the rest.
```